ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -72,9 +72,11 @@
     private static final By MERGE_VISIT_BUTTON = By.xpath("//a[@id='org.openmrs.module.coreapps.mergeVisits']/li");
     private static final By MERGE = By.id("mergeVisitsBtn");
     private static final By PATIENT = By.xpath("//ul[@id='breadcrumbs']/li[2]/a");
+    private static final By EYE_REPORT = By.linkText("Eye Report");
```

Comment:
Should we not get a better element reference since `linkText` is not reliable? How about using `cssSelector`, or id or `xPath`

Reply:
Actually this should be removed we noloner support eye report

Reply:
> Eye report is no longer supported as you have been seeing different conversations we normally had

But git is tracking this as your change. If its not true then I recommend you do some settings in your IDE for git only to reflect your changes!

Reply:
i wouldnt to touch on this file, i would like to leave as it is since  eye ReportTest class is still in the master

Reply:
Unless we clearly remove it which can be worked on a different PR.

Reply:
> Unless we clearly remove it which can be worked on a different PR.

Are you sure that its not used by some instances ?. If no we can still use a different element look up to ensure code sustainability otherwise `linkText` is not smart enough

Reply:
> i wouldnt to touch on this file, i would like to leave as it is since eye ReportTest class is still in the master

Do you mind sharing a link to this file ?  

BTW  did you see my recommendation [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/382#discussion_r682728044) !